repose-aggregator/tests/functional-tests/src/integrationTest/configs/features/filters/keystonev2/authorizationstandalone/common/system-model.cfg.xml

```diff
@@ -5,6 +5,7 @@
             <node id="simple-node" hostname="localhost" http-port="${reposePort}"/>
         </nodes>
         <filters>
+            <filter name="url-extractor-to-header" />
```

Comment:
Oh. Should I not tell you that we do this... a... bunch...?

This has always been in my not-worth-the-effort category since we only do it in our test configurations. If you want it changed, though, it's an easy find-and-replace.